Kros.KORM/tests/Kros.KORM.UnitTests/Migrations/Providers/AssemblyMigrationScriptsProviderShould.cs

```diff
@@ -0,0 +1,107 @@
+﻿using FluentAssertions;
+using Kros.KORM.Migrations.Providers;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using System.Text;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace Kros.KORM.UnitTests.Migrations.Providers
+{
+    public class AssemblyMigrationScriptsProviderShould
+    {
+        [Fact]
+        public void GetScriptsFromDefaultNamespace()
+        {
+            string GetNamespace(string fileName)
+                => $"Kros.KORM.UnitTests.SqlScripts.{fileName}.sql";
+
+            AssemblyMigrationScriptsProvider provider = CreateDefaultProvider();
+            var scripts = provider.GetScripts().ToList();
+
+            scripts.Count.Should().Be(3);
```

Comment:
- We do not need assertion for `Count`, because it is _included_ in `Equals` line below. This is in `GetScriptFromDefinedAsemblyAndNamespace` test as well.
- And if you need to check `Count`, there is `collection.Should().HaveCount(n)` - to be consistent with other checks (and it also supports an expression).